public/css/admin.css

```diff
@@ -42,4 +42,12 @@
 
 .alignTextLeft {
     text-align: left;
+}
+
+.hiddenEye {
+    color: rgba(238, 110, 115, 1);;
```

Comment:
There are 2 `;` here. Also since you have the opacity as 1, please remove it and keep this colour as `rgb`

Reply:
fixed